pkg/parse/parse.go

```diff
@@ -607,29 +612,36 @@ func GetBuildOutput(buildOutput string) (define.BuildOutputOption, error) {
 				return define.BuildOutputOption{}, fmt.Errorf("duplicate %q not supported", arr[0])
 			}
 			typeSelected = true
-			if arr[1] == "local" {
-				isDir = true
-			} else if arr[1] == "tar" {
-				isDir = false
-			} else {
+			switch arr[1] {
+			case "local":
+				out.IsDir = true
+			case "tar":
+				out.IsDir = false
+			case "registry":
+				out.IsDir = true
```

Comment:
This is surprising; at the _very least_ it should be documented in the field, or have a comment here explaining why it is necessary. (But my starting hypothesis is that the data structure should be redesigned instead, to make this unnecessary — I’m not sure whether there are any compatibility impacts.)

Reply:
This wasn't necessary, I think I forgot to clean this up as part of the recent refactor.